proto/internal/temporal/server/api/batch/v1/request_response.proto

```diff
@@ -22,4 +23,7 @@ message BatchOperationInput {
 
   // The request to start the batch operation.
   temporal.api.workflowservice.v1.StartBatchOperationRequest request = 7;
+
+  // The request to start an admin batch operation.
```

Comment:
plz comment that this field and `request` are mutually exclusive.
plz also add a comment for batch_type and say that it only applies to `request`